packages/theme-a11y/theme-a11y.puppeteer.test.js

```diff
@@ -3,9 +3,9 @@ import {
   focusable,
   trapFocus,
   removeTrapFocus
-} from './theme-a11y';
+} from "./theme-a11y";
```

Comment:
Can you set `"prettier.singleQuote": true,` in your VSCode config so these stay as `'`

Reply:
I'll create a separate PR to include Eslint config.